### ISSUE TITLE
Malformed ZIP files aren't extracted correctly on Linux

### ISSUE THREAD

**Issue by notpeelz** [Kesomannen/gale#101]
Since 2af5378f9ef274e065eef379efddbd0c175423b8, [BarchLib](https://thunderstore.io/c/lethal-company/p/qwbarch/BarchLib/) doesn't install correctly because its ZIP contains [non-standard file path separators](https://stackoverflow.com/a/60276958/1581233).

```
.
├── BepInEx\core\FSharp.Control.AsyncSeq.dll
├── BepInEx\core\FSharp.Core.dll
├── BepInEx\core\FSharpPlus.dll
├── BepInEx\core\FSharpx.Async.dll
├── BepInEx\core\libmp3lame.32.dll
├── BepInEx\core\libmp3lame.64.dll
├── BepInEx\core\NAudio.Asio.dll
├── BepInEx\core\NAudio.Core.dll
├── BepInEx\core\NAudio.dll
├── BepInEx\core\NAudio.Lame.dll
├── BepInEx\core\NAudio.Midi.dll
├── BepInEx\core\NAudio.Wasapi.dll
├── BepInEx\core\NAudio.WinForms.dll
├── BepInEx\core\NAudio.WinMM.dll
├── BepInEx\core\UniTask.Addressables.dll
├── BepInEx\core\UniTask.dll
├── BepInEx\core\UniTask.DOTween.dll
├── BepInEx\core\UniTask.Linq.dll
├── BepInEx\core\UniTask.TextMeshPro.dll
├── BepInEx\core\WebRtcVad.dll
├── BepInEx\core\WebRtcVad.pdb
├── BepInEx\core\WebRtcVadSharp.dll
├── BepInEx\core\WebRtcVadSharp.pdb
├── icon.png
├── manifest.json
└── README.md
```

This results in filenames with literal backslashes in the `BepInEx/plugins` folder.

**Comment by SolakkaSilakka** [comment]
Here's a screenshot of how it looks like on my end.
![kuva](https://github.com/user-attachments/assets/c5bfa892-8523-477b-bb7c-54a2d695f4d3)

**Comment by gusruben** [comment]
Can confirm, this happens to me too. The game runs fine if I manually extract/move the files to the right place.

**Comment by Kesomannen** [comment]
Well, that's dumb

**Comment by Kesomannen** [comment]
Should be fixed in 0.8.3

**Comment by notpeelz** [comment]
Appears to be fixed. I just had to clear the cache in order to fix the misplaced files.
